app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,38 @@
+class StatsController < ApplicationController
+  include ActionController::HttpAuthentication::Token::ControllerMethods
+
+  before_action :authenticate, :set_access_control_headers
+
+  def index
+    @audited_publications = PublicActivity::Activity.where(key: 'vacancy.publish')
```

Comment:
Could we display the 2 stats that are driving this change along with all counts with a small change to the following: `PublicActivity::Activity.group(:key).count`? This means as soon as we start tracking a future activity, they are automatically included without having to make an additional change to this page.